src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -61,7 +48,7 @@ export function uiRenderMixin(kbnServer, server, config) {
           allDefaults,
           fn(kbnServer.server, pluginSpec.readConfigValue(kbnServer.config, []))
         )
-      ), getInitialDefaultInjectedVars());
+      ), { uiCapabilities: { navLinks: {} } });
```

Comment:
Can we get rid of this entirely, now what we aren't using ui capabilities from the injected vars?

Reply:
Good catch, thanks!